lib/test/benchpark/directives.py

```diff
@@ -0,0 +1,4 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
```

Comment:
Does this do anything?

Reply:
Not that I know of: it came from https://github.com/LLNL/benchpark/pull/299/files#diff-ce8ab53a2b3008e27988e961de694d655e49cc693f625e6d82c19a437715cb19